easybuild/easyblocks/t/tensorflow.py

```diff
@@ -760,6 +771,14 @@ def build_step(self):
         self.bazel_opts = [
             '--output_user_root=%s' % self.output_user_root_dir,
         ]
+        if bazel_version >= '4.0.0':
+            self.bazel_opts.append('--local_startup_timeout_secs=600')  # 5min for bazel to start
```

Comment:
Can you clarify why this is needed?
What's taking Bazel 4.x+ so long to "start"?

Reply:
The option was introduced in Bazel 4.0. I was observing failures on some of our nodes where the build failed (after installing all 20 or so extensions) due to a timeout of 2min "connecting to Bazel"
As it could be a filesystem issue being slow to read new stuff I wanted to use the new flag.

I enhanced the comment and made it actually 5min, not 10 ;-)